src/api/docs/doc.test.js

```diff
@@ -1,10 +1,10 @@
 const puppeteer = require('puppeteer');
-const { PORT } = require('../../config');
+const { HOST, PORT } = require('../../config');
 const app = require('../../app').listen(PORT);
 
 let browser;
 let page;
-const BASE_URL = `http://localhost:${PORT}`;
+const BASE_URL = `http://${HOST}:${PORT}`;
```

Comment:
You are right. It wasn't necessary.

In fact, it is necessary that the host is `localhost` because of the 3rd test. I was thinking about this in the opposite direction.